cmd/tailscale/cli/update.go

```diff
@@ -44,17 +44,19 @@ var updateCmd = &ffcli.Command{
 		fs := newFlagSet("update")
 		fs.BoolVar(&updateArgs.yes, "yes", false, "update without interactive prompts")
 		fs.BoolVar(&updateArgs.dryRun, "dry-run", false, "print what update would do without doing it, or prompts")
+		fs.BoolVar(&updateArgs.appStore, "app-store", false, "check the App Store for updates, even if this is not an App Store install (for testing only!)")
```

Comment:
```suggestion
		fs.BoolVar(&updateArgs.appStore, "app-store", false, "check the Mac App Store for updates, even if this is not an App Store install (for testing only!)")
```

Reply:
I imagine we might want to overload this for testing on Windows, too. Having separate options for Mac and Win, but each only used on 1 platform, would be overly verbose.